src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx

```diff
@@ -122,494 +122,153 @@ Here's how you can access the distributed tracing UI, depending on the type of s
+                title="Screenshot showing left-pane filters"
+                alt="Screenshot showing left-pane filters"
+                src={distributedtracingLeftPaneFilters}
+            />
+              * **Infinite tracing data**: Select this only to show traces related to the Infinite Tracing feature.
+              * **Multi span traces only**: Use this to hide single-span traces.
+              * **Error filters**: Under **Errors** select the errors to filter by.
+              * **Histogram filters**: Below **Errors** at the bottom of the left pane, you can click **More filters** to show histogram filters. Drag the sliders to change the values, such as **Trace duration**:
+                * Drag the left end of the slider for greater-than comparisons.
+                * Drag the right end of the slider for less-than comparisons.
+                * Drag each end of the slider toward the center to filter by a range.
+
+                <img
+                  title="new-relic-one-distributed-tracing-histogram-selection.png"
+                  alt="Distributed tracing - histogram"
+                  src={newRelicOneDistributedTracingHistogramSelection}
+                />
+                <figcaption>
+                  When you drag the slider, it changes both the list of traces and what's shown in the trace charts.
+                </figcaption>
+        </TabsPageItem>
+
+        <TabsPageItem id="3">
+            The default view of distributed tracing shows traces grouped by the same root entry span. In other words, traces are grouped by the span where New Relic began recording the request. You can slide the toggle **Group similar traces** to turn this on and off.
+
+            With trace groups you get a high-level view of traces so you can understand request behavior for groups of similar traces. This helps you understand dips or spikes in trace count, duration, and errors. When you click on one of the trace groups, you get all the standard details in context of the specific trace group you selected.
+
+            You can control what you see in the trace groups list by using left-pane filters.
+        </TabsPageItem>
+        <TabsPageItem id="4">
+            The trace scatter plot is a quick way to search for outlying traces. This is available on the opening page of distributed tracing if you turn off the **Group similar traces** toggle at the top of the page.
+
+            In the scatter plot, you can move the cursor across the chart to view trace details and you can click individual points to get details:
+
+            <img
+              title="new-relic-one-distributed-tracing-histogram-selection.webp"
+              alt="Screenshot showing the distributed tracing scatter plot chart."
+              src={newRelicOneDistributedTracingScatterplot}
+            />
+
+            Here's how you can control what's displayed in the scatter plot:
+
+            1. In the **View by** dropdown, select the duration type:
```

Comment:
I'd recommend leading steps with an active verb and swapping the clauses here (e.g., 'Select duration type in the **View by** dropdown')